src/pages/hyeonjooKim/Main/FeedComponent/Feed.js

```diff
@@ -0,0 +1,85 @@
+import { useState, useEffect } from 'react';
+import Reply from './Reply';
+import InputReply from './InputReply';
+import './Feed.scss';
+
+const Feed = ({ id, userName, content }) => {
+  const [reply, setReply] = useState([]);
+  const [replyId, setReplyId] = useState(0);
+
+  const saveReply = inputValue => {
+    setReply(prevState => {
+      return [
+        ...prevState,
+        {
+          id: replyId,
+          userName: 'hj_kim',
+          content: inputValue,
+        },
+      ];
+    });
+    setReplyId(prevState => prevState + 1);
+  };
+
+  const deleteReply = id => {
+    setReply(reply.filter(ele => ele.id !== id));
+  };
+
+  // mock data를 활용한 댓글 입력 확인
+  useEffect(() => {
+    fetch('http://localhost:3000/data/commentDataHj.json')
+      .then(res => res.json())
+      .then(data => {
+        setReply(data);
+      });
```

Comment:
```suggestion
    fetch('/data/commentDataHj.json')
      .then(res => res.json())
      .then(data => {
        setReply(data);
      });
```
목데이터로 fetch를 사용하는 경우에는 `localhost:3000` 생략이 가능합니다